src/utility/sendCount.js

```diff
@@ -0,0 +1,49 @@
+const Axios = require('axios').default;
+
+/**
+ * An module to auto send your server counts to botlists.com
+ * @param {*} client Your Discord Bot Client
+ * @param {String} token Your Bot's Token for Website API
+ * @param {Number} timeout Timeinterval between sending server count
+ * @param {Boolean} disableConsole Whether you want console logs or not
+ */
+module.exports = (client, token, timeout = 1800000, disableConsole = false) => {
+    return new Promise((res, rej) => {
+        try {
```

Comment:
I don't get the try/catch block here if you're using .then and .catch.

Reply:
fixed